tests/telemetry/test_telemetry_metrics.py

```diff
@@ -365,7 +386,7 @@ def test_send_log_metric_simple_tags(telemetry_metrics_writer, test_agent_metric
                 "level": "WARNING",
                 "message": "test error 1",
                 "tracer_time": 1642544540,
-                "tags": "a:b,c:d",
+                "tags": ["a:b", "c:d"],
```

Comment:
Metrics tags are reported as a list of strings "{key}:{value}" but LOGS tags are reported as a comma separated string "{key}:{value},{key}:{value},{key}:{value}" :+1: